store/storetest/channel_store.go

```diff
@@ -1039,8 +1038,7 @@ func testChannelStoreGetChannels(t *testing.T, ss store.Store) {
 	m3.NotifyProps = model.GetDefaultChannelNotifyProps()
 	store.Must(ss.Channel().SaveMember(&m3))
 
-	cresult := <-ss.Channel().GetChannels(o1.TeamId, m1.UserId, false)
-	list := cresult.Data.(*model.ChannelList)
+	list, _ := ss.Channel().GetChannels(o1.TeamId, m1.UserId, false)
```

Comment:
I would expect to add here a `require.Nil(t, err)` because we are de-referencing that list in the next line of code.

Reply:
Ok, I added `require.Nil(t, err)`